proof_of_concept.R

```diff
@@ -89,19 +89,22 @@ record_ids <- seq(first_id, first_id + number_of_records_to_populate)
 # get the categorical field responses in a long table and populate them
 long_categorical_field_responses <- get_long_categorical_field_responses(metadata_to_populate)
 long_text_fields <- get_long_text_fields(metadata_to_populate)
+long_notes_fields <- get_long_notes_fields(metadata_to_populate)
 
 long_fields_and_responses <- bind_rows(
   long_categorical_field_responses,
-  long_text_fields
+  long_text_fields,
+  long_notes_fields
 )
 
 picked_values <-
-  purrr::map(record_ids,
-             get_one_rectangle_of_values,
-             record_id_name,
-             forms_to_fill,
-             long_fields_and_responses
-             ) |>
+  purrr::map(
+    record_ids,
+    get_one_rectangle_of_values,
+    record_id_name,
+    forms_to_fill,
+    long_fields_and_responses
+  ) |>
```

Comment:
Thank you for restyling. I don't know how we missed this. Styler has a "restyle package" option. I thought I was using it religiously, but apparently not. :-D